meinberlin/templates/a4modules/includes/module_detail_phase.html

```diff
@@ -19,25 +19,36 @@
 {% endif %}
 {% with phase_count=module.phases.count %}
 <div class="offset-lg-3 col-lg-6">
-    {% for phase in module.phases %}
-        <div class="phase-info__item {% if phase_count == 1 %} u-no-border {% endif %} u-spacer-top">
-            <div class="phase-info__item__title{% if phase == module.active_phase %} lr-bar u-inline-flex{% endif %}">
-                {% if phase == module.active_phase %}
-                    <span class="lr-bar__left">{{ phase.name }}</span>
-                    <span class="lr-bar__right phase-info__highlight">{% trans 'active' %}</span>
-                {% else %}
-                    {{ phase.name }}
-                {% endif %}
-            </div>
-            <div class="phase-info__item__subtitle">
-                {% html_date phase.start_date 'DATETIME_FORMAT' %}
-                &ndash; {% html_date phase.end_date 'DATETIME_FORMAT' %}
-            </div>
-            <div class="phase-info__item__description">
-                {{ phase.description }}
-            </div>
+    <!-- Start Swiper Template -->
+    <div class="swiper-phases swiper">
```

Comment:
only using `swiper-phases` caused problem, so in addition it needs `swiper` to make it work properly.

Reply:
@khamui having spoken to @fuzzylogic2000 we do not want to load this when it is a single phase module so we should use a separate include for single phase modules and have this as an include for multi phase modules

Reply:
yes makes sense, will do that